component-template/{{ cookiecutter.slug }}/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [v0.1.0] - {{ cookiecutter.release_date }}
```

Comment:
Not sure I like having a `v0.1.0` in the template, but if we keep it, we should consider immediately tagging v0.1.0 on the initial commit.

Reply:
Also not a big fan. Let's only keep `Unreleased`.

Reply:
I tried to find a way to showcase how the change log is supposed to be used. My assumption is, if left out, the change log will become everything but consistent. Also I was unable to find a compare link comparing the first commit with HEAD without knowing the commit ID of that first commit. Immediately tagging that first commit, seems to be the cleanest option. Probably will go with `v0.0.1` instead.

Reply:
I like v0.0.1 for the initial commit much better :)

Reply:
Discussed this this morning in our daily. We agreed on removing that initial version keeping only the _Unrelease_. We also keep the link for unreleased using the tag `v0.1.0` as the comparison base. We accept the fact, that this will be an invalid link until a first version will be released.